docs/appkit/shared/socials.mdx

```diff
@@ -73,7 +73,7 @@ const modal = createAppKit({
 ## Options
 
 - **_email [boolean]_** : This boolean defines whether you want to enable email login. Default `true`
-- **_socials [array]_** : This array contains the list of social platforms that you want to enable for user authentication. The platforms in the example include Google, X, GitHub, Discord, Apple, Facebook and Farcaster. Default `undefined`
+- **_socials [array]_** : This array contains the list of social platforms that you want to enable for user authentication. The platforms in the example include Google, X, GitHub, Discord, Apple, Facebook and Farcaster. The default value of `undefined` displays everything. Set it to `false` to disable this feature.
```

Comment:
This is incorrect by the way, @rtomas. 

We need to pass an empty array to disable it. I don't think you can set it to `false` since it isn't of boolean data type. Can you please double check?